ibis/config.py

```diff
@@ -203,7 +203,9 @@ def _default_backend() -> Any:
 
     import ibis
 
-    options.default_backend = con = ibis.duckdb.connect(":memory:")
+    options.default_backend = con = ibis.duckdb.from_connection(
+        duckdb.default_connection
```

Comment:
What apis in duckdb result in their default connection being used when using duckdb alone? Is there any risk in us sharing this connection with other things a user might be using?

Reply:
Yeah, perhaps.

I'll close the PR, since the gain isn't worth having to think about who owns the connection.